### PR TITLE
fix: ui feedbacks

### LINKED ISSUE #12363: 1.1 UI - Feedbacks
**Affected module**
UI

**Describe the bug**
Congratulations on the new UI! 🥇

I have a few inquiries and comments regarding the new user interface and user experience.

1 - The empty space in "explore" page, it's normal or a bug?
Capture d’écran 2023-07-10 à 15 59 42

2- When the column is PII, the column type is not reported, i think removing the ()  is better than having it!
Capture d’écran 2023-07-10 à 16 01 52

3- When pagination is present, each click on 'Next' or 'Previous' causes the entire block to reload. This includes both the database table description and the HTML table structure, making the user interface less robust. The anticipated behavior should be such that only the data within the HTML table gets updated in the user interface.

Capture d’écran 2023-07-10 à 16 04 22

**To Reproduce**

Screenshots or steps to reproduce

**Expected behavior**

**Version:**
 - Python version:
 - OpenMetadata version: [e.g. 0.8]
 - OpenMetadata Ingestion package version: [e.g. `openmetadata-ingestion[docker]==XYZ`]

**Additional context**

### LINKED ISSUE #12379: UI: Description re-renders on clicking Next
**Affected module**
Does it impact the UI, backend or Ingestion Framework?
UI

**Describe the bug**
Description component re renders on clicking next
**To Reproduce**
- go to the [databaseschema/ table page](https://sandbox-beta.open-metadata.org/databaseSchema/sample_data.ecommerce_db.shopify)
- Click on Next / Previous

Screenshots or steps to reproduce

https://github.com/open-metadata/OpenMetadata/assets/40225091/b5db7e20-110f-4e68-b73a-1f331811616b

**Expected behavior**
- The Description for a table should stay rendered whereas the next tables should render

**Version:**
 - Python version:
 - OpenMetadata version: [e.g. 0.8]
 - OpenMetadata Ingestion package version: [e.g. `openmetadata-ingestion[docker]==XYZ`]

**Additional context**

### PR DESCRIPTION
**Pull request by karanh37**
### Describe your changes:

Fixes UI feedbacks
Fixes #12379 
Fixes #12363 

I worked on ... because ...

#
### Type of change:

- [x] Bug fix

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/database-schema-page.less [hunk 1]
```diff
@@ -10,6 +10,8 @@
...
  *  limitations under the License.
  */
-
-export const activeLink = '#000000';
-export const normalLink = '#37352f';
+.database-schema-page-tabs {
+  .ant-tabs-nav {
+    margin: 0 !important;
+  }
+}
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/database-schema-page.less:17]: Can we avoid this

File: openmetadata-ui/src/main/resources/ui/src/pages/EmailConfigSettingsPage/EmailConfigSettingsPage.component.tsx [hunk 2]
```diff
@@ -108,15 +108,12 @@ function EmailConfigSettingsPage() {
     }
 
     return (
-      <Card>
+      <Card className="card-body-border-none card-padding-0">
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/EmailConfigSettingsPage/EmailConfigSettingsPage.component.tsx:111]: remove card

File: openmetadata-ui/src/main/resources/ui/src/pages/KPIPage/EditKPIPage.tsx [hunk 3]
```diff
@@ -245,198 +244,196 @@ const EditKPIPage = () => {
...
+                  type="link"
+                  onClick={handleCancel}>
+                  {t('label.go-back')}
+                </Button>
+                <Tooltip
+                  title={
+                    isAdminUser
+                      ? t('label.save')
+                      : t('message.no-permission-for-action')
+                  }>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/KPIPage/EditKPIPage.tsx:424]: We no longer need this as we will be hiding actions if no permission present

File: openmetadata-ui/src/main/resources/ui/src/pages/KPIPage/EditKPIPage.tsx [hunk 4]
```diff
@@ -245,198 +244,196 @@ const EditKPIPage = () => {
...
                   <Button
-                    data-testid="cancel-btn"
-                    type="link"
-                    onClick={handleCancel}>
-                    {t('label.go-back')}
+                    data-testid="submit-btn"
+                    disabled={!isAdminUser}
+                    form="kpi-form"
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/KPIPage/EditKPIPage.tsx:428]: We don't need form prop if we have set htmlType="submit"

File: openmetadata-ui/src/main/resources/ui/src/pages/KPIPage/AddKPIPage.tsx [hunk 5]
```diff
@@ -229,227 +228,224 @@ const AddKPIPage = () => {
...
-                  name="metricType"
+                  label={t('label.metric-value')}
+                  name="metricValue"
                   rules={[
                     {
                       required: true,
-                      message: t('message.field-text-is-required', {
-                        fieldText: t('label.metric-type'),
-                      }),
+                      validator: () => {
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/KPIPage/AddKPIPage.tsx:311]: we can use min max for this validation i believe

File: openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/database-schema-page.less (hunk 1 above)
Comment (karanh37) [reply openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/database-schema-page.less:17]: No we need this. Antd internally applied margin

File: openmetadata-ui/src/main/resources/ui/src/pages/KPIPage/AddKPIPage.tsx (hunk 5 above)
Comment (karanh37) [reply openmetadata-ui/src/main/resources/ui/src/pages/KPIPage/AddKPIPage.tsx:311]: can we take care of this later ?

File: openmetadata-ui/src/main/resources/ui/src/pages/AddAlertPage/AddAlertPage.tsx [hunk 6]
```diff
@@ -497,222 +497,237 @@ const AddAlertPage = () => {
...
+                  name="description"
+                  trigger="onTextChange"
+                  valuePropName="initialValue">
+                  <RichTextEditor
+                    data-testid="description"
+                    height="200px"
+                    initialValue=""
+                  />
+                </Form.Item>
+                <Form.Item>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/AddAlertPage/AddAlertPage.tsx:529]: Form.Item seems redundant

File: openmetadata-ui/src/main/resources/ui/src/pages/AddAlertPage/AddAlertPage.tsx [hunk 7]
```diff
@@ -497,222 +497,237 @@ const AddAlertPage = () => {
...
+                                  block
+                                  data-testid="add-filters"
+                                  icon={<PlusOutlined />}
+                                  type="default"
+                                  onClick={() => add({}, 0)}>
+                                  {t('label.add-entity', {
+                                    entity: t('label.filter-plural'),
+                                  })}
+                                </Button>
+                              </Form.Item>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/AddAlertPage/AddAlertPage.tsx:588]: Form.Item for button is not required

File: openmetadata-ui/src/main/resources/ui/src/pages/AddAlertPage/AddAlertPage.tsx [hunk 8]
```diff
@@ -497,222 +497,237 @@ const AddAlertPage = () => {
...
+                    <Col span={8}>
+                      <Space className="w-full" direction="vertical" size={16}>
+                        <StyledCard
+                          heading={t('label.destination')}
+                          subHeading={t(
+                            'message.alerts-destination-description'
+                          )}
+                        />
+                        <Form.Item>
+                          <Form.Item
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/AddAlertPage/AddAlertPage.tsx:676]: twice form.item seems wrong here

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**